codepipeline-tf/providers.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
+  required_version = ">= 1.0"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 5.0"
+    }
+  }
+}
+
+provider "aws" {
+  region = "eu-west-1"

```

Comment:
@kashan-01 We should make the region variable. And if we need eu-west-1 as default we can set it in variables.tf

Reply:
Done